internal/ui/config.go

```diff
@@ -195,7 +195,14 @@ func (c *Configurator) activeSkin() (string, bool) {
 		return skin, false
 	}
 
-	if ct, err := c.Config.K9s.ActiveContext(); err == nil && ct.Skin != "" {
+	if env_skin := os.Getenv("K9S_SKIN"); env_skin != "" {
```

Comment:
if the skin env var is set and valid I think we should just return since it overrides any configs skins anyway.